types/wordpress__components/package.json

```diff
@@ -2,6 +2,7 @@
     "private": true,
     "dependencies": {
         "@wordpress/element": "^2.14.0",
+        "downshift": "^5.4.0",
```

Comment:
It does seem to bundle typings:

https://unpkg.com/browse/downshift@5.0.4/typings/index.d.ts

This version aligns with the latest `@wordpress/components` release:

https://unpkg.com/browse/@wordpress/components@11.0.0/package.json

Reply:
The version header for the index.d.ts for this package is `// Type definitions for @wordpress/components 9.8`. The version of downshift used for v9.8 was `^4.0.5`. I would recommend either:

- Using the typings from downshift@4.0.5, or
- Updating the version header in index.d.ts